src/main/java/com/kuljava/swiatwsi/rawmaterials/RawMaterials.java

```diff
@@ -0,0 +1,34 @@
+package com.kuljava.swiatwsi.rawmaterials;
+
+import com.kuljava.swiatwsi.world.Village;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+import lombok.ToString;
+
+import javax.persistence.*;
+import java.io.Serializable;
+
+@Entity
+@Table(name = "raw_materials")
```

Comment:
Is name attribute needed here? Probably Hibernate would create table for this entity with this name without this attribute.

Reply:
Yes, but why not? I prefer to use '_' in database table and columns names. Is it mistake?

Reply:
Yeah but hibernate is smart enough to know that. He will convert camelCase to name with _